src/studio-home/tabs-section/libraries-v2-tab/index.tsx

```diff
@@ -98,7 +98,7 @@ const LibrariesV2Tab = ({
         </div>
 
         { hasV2Libraries
-          ? data.results.map(({
+          ? data?.results.map(({
```

Comment:
react-query guarantees that only one of `data`, `isLoading`, or `isError` will be truthy at any given time, so you can either change this to `data!.results` or refactor the code like this:

```ts
if (isLoading) {
  // Display loading state
}
if (isError || data === undefind) {
  // Display an error message. The `|| data === undefined` is only needed for TypeScript to understand the flow.
}

// Render the normal component. Here TypeScript now knows that `data` is not undefined
```

Reply:
I see, I've updated it to use the `data!` and made sure that the to check for `!isLoaded` in the `hasV2Libraries` which is used in a bunch of places. (`isError` is already checked at the beginning and returns): [a72713e](https://github.com/open-craft/frontend-app-course-authoring/pull/43/commits/a72713e34decb07215a236f85ff70d5d6ff9f21e)